drivers/modem/Kconfig.cellular

```diff
@@ -55,4 +57,16 @@ config MODEM_CELLULAR_USER_PIPE_BUFFER_SIZES
 	int "The size of the buffers used for each user pipe in bytes."
 	default 128
 
+config MODEM_CELLULAR_NEW_BAUDRATE
+	int "New baudrate to configure modem to, if supported"
+	range 0 4000000
```

Comment:
a range of 0 doesn't make sense does it? I recommend setting the min to something like 9600.

Reply:
fixed